mutt/file.c

```diff
@@ -930,11 +930,11 @@ time_t mutt_file_decrease_mtime(const char *f, struct stat *st)
  *
  * @warning mutt_file_dirname() returns a static string which must not be free()'d.
  */
-const char *mutt_file_dirname(const char *p)
+char *mutt_file_dirname(const char *p)
 {
-  static char buf[_POSIX_PATH_MAX];
+  char *buf = mutt_mem_malloc(sizeof(_POSIX_PATH_MAX));
```

Comment:
`sizeof(_POSIX_PATH_MAX)` == 4
You just want `_POSIX_PATH_MAX`

Reply:
Thank you again!